presto/tests/test_presto.py

```diff
@@ -0,0 +1,11 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+
+## We don't test JMXFetch based integrations in this repo
+## This is required to allow E2E to spin up the environment
+@pytest.mark.usefixtures('dd_environment')
```

Comment:
@ofek Do we need this to spin up the env? Seems odd to me.

Reply:
Hey @gzussa! Since java and jmxfetch isn't packaged with the standard agent images, you'll need to start the docker environment using the `-jmx` tag of agent image. And `ddev env check` doesn't seem to work with jmx checks, it just hangs. @ofek, @nmuesch, and I were noticing that while testing last week. Not exactly sure where the limitation is in the code. 

Try `ddev env start presto py27-latest -a datadog/agent:latest-jmx` 

That should start the presto docker environment with the jmx agent. Although we'll still need to fix `ddev check` to work with jmx checks to fully be able to run the check against the docker environment.

Reply:
Yes, for now